### ISSUE TITLE
Is there a way to get this script to download television library posters?

### ISSUE THREAD

**Issue by scottruzal** [CTinMich/Plex-DB-Movie-Posters-Downloader#1]
Thank you for putting this together! I had no trouble using this to download movie library posters from PMS for Windows 10.

Looking at your script and then looking through Plex's metadata for Television libraries, it seems like there is a bunch of parent and child metadata to get through if I wanted to use this to download show and season posters for a Television library. 

My knowledge of Python is very rudimentary, and I was wondering if you might already have a separate script for downloading TV posters, or if you might have any recommendations on how best to go about it if I were to rework this myself.

Would greatly appreciate any help you might be able to provide!

**Comment by CTinMich** [comment]
Hey Scott...

I just updated the script to do TV also.  But....  it is not the cleanest because looking at the TV Show library with the Plex API it does not show the "path" for TV Shows Library like the Movies Library does.  So...  I updated the code to assume that the Show title is the beginning of the folder name (renaming some special character (":" to " -") and stripping others ("?").  I based that off how "Filebot" handles special characters in it's renames process.  

I also added some command line arguments to allow specify which library (Move/TV) to modify and overwrite existing posters.
